packages/web-components/fast-components/src/toolbar/toolbar.styles.ts

```diff
@@ -69,6 +69,11 @@ export const toolbarStyles: (
             margin-inline: 0;
         }
 
+        .start__hidden,
+        .end__hidden {
+            display: none;
+        }
```

Comment:
Can you elaborate on why these are necessary to add as well as why they are *defaulted* and *hardcoded* into the template?

Reply:
I added these so that the start and end don't show up at all, instead of being a 0x0 span when nothing is slotted. Is either way preferred or does it not really matter since they are a 0x0 div?

To get them to show hidden from the start they were added to the template, but  `this.startSlottedItemsChanged()` and `this.endSlottedItemsChanged();` could be added to the connectedCallback to run at the start, which would then toggle the __hidden classes as needed.

Would you recommend I remove all the __hidden and related code if the 0x0 span is an acceptable result?

Reply:
removed __hidden and related code, not necessary